src/source/PeerConnection/PeerConnection.c

```diff
@@ -1563,6 +1563,10 @@ STATUS addTransceiver(PRtcPeerConnection pPeerConnection, PRtcMediaStreamTrack p
             depayFunc = depayVP8FromRtpPayload;
             clockRate = VIDEO_CLOCKRATE;
             break;
+        case RTC_CODEC_H265:
```

Comment:
Do we need to specify a more descriptive name here that includes the profile / packetization mode like we have for h264?

Reply:
That's because there is a default fmtp we use for h264 which is not the case for h265 as nothing has been enforced in the RFCs as far as I checked last. The offers that I received from Safari and Android do not contain fmtp / profile information either so I think it is more appropriate to use the current name

Reply:
ack